Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1

```diff
@@ -306,7 +305,8 @@ function Invoke-NewDevelopmentAppSettings([hashtable] $Settings = @{ }) {
 
         Write-Host
         Write-Host 'initdev is now using the following settings:' -ForegroundColor Green
-        Write-FlatHashtable (Get-DeploymentSettings)
+        $DeploymentSettings= Merge-Hashtables (Get-DeploymentSettings),(Display-UserSecretsList )
```

Comment:
since most of the initdev tasks call `Get-DeploymentSettings` to get their settings I don't see how merging it here is going to work. As discussed this needs to be moved into that function with **only** the user secrets from the webapi project.